PluginInstaller/Create-PluginInstaller.ps1

```diff
@@ -122,18 +79,19 @@ function main {
     }
 
     function prep-template {
-      $contents = $template
-
-      if ( $upgrade_code -match "$" ) { $upgrade_code = Invoke-Expression ($upgrade_code) }
-
-      $contents = $contents.Replace($default_title_cab, $title.Replace(" ", "_"));
```

Comment:
Why change the way in which the title/etc are replaced?  Again, seems un-necessary?